src/components/formatting/IRSIndicatorLegend/tests/index.test.tsx

```diff
@@ -44,4 +49,18 @@ describe('/components/formatting/IRSIndicatorLegend', () => {
       { color: '#2ECC40', name: 'Green', orEquals: true, text: '90% - 100%', value: 1 },
     ]);
   });
+
+  it('renders correctly for namibia indicator colors with namibia indicatorThresholds', () => {
+    const wrapper = mount(
+      <IRSIndicatorLegend
+        indicatorRows={namibiaIndicatorRows}
+        indicatorThresholds={indicatorThresholdsIRSNamibia}
+      />
+    );
+    expect(wrapper.props()).toEqual({
+      indicatorRows: namibiaIndicatorRows,
+      indicatorThresholds: indicatorThresholdsIRSNamibia,
+      indicatorThresholdsLookUp: indicatorThresholdsLookUpIRS,
+    });
+  });
```

Comment:
@ciremusyoka What are we testing here, seems like we are checking whether the props we pass to the component are the ones that the component recieves, wierd isn't it. On the flip side i think we already have test for NamibiaIRS ithink at line l28.

Reply:
@p-netm here we a passing on `indicatorThresholds` which is an optional prop but for the previous test we were not passing the optional parameter.

Reply:
@ciremusyoka so at a high level what would you describe that this test is testing,

Reply:
@p-netm have resolved this